devices/HapticGlove/src/HapticGlove.cpp

```diff
@@ -0,0 +1,711 @@
+﻿/**
+ * @file HapticGlove.cpp
+ * @authors  Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#include <HapticGlove.h>
+#include <SenseGloveHelper.hpp>
+
+#include <yarp/os/Bottle.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Time.h>
+
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/RpcServer.h>
+
+#include <assert.h>
+#include <map>
+#include <mutex>
+#include <stdio.h>
+#include <string>
+#include <vector>
+
+const std::string DeviceName = "HapticGlove";
+const std::string LogPrefix = DeviceName + wearable::Separator;
+double period = 0.01;
```

Comment:
Is this the default period or it can also be configured?

Reply:
This is the default value, later it is updated witht the value read from the xml file.